doc/short_tutorial.rst

```diff
@@ -0,0 +1,169 @@
+.. SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _short_tutorial:
+
+
+##########################################
+Short tutorial
+##########################################
+
+The installation procedure installs PyPSA-Distribution model with all the software dependencies needed to build and run it.
+It is first crucial to get familiar with a tutorial where a simpler model is considered. This section explains how to:
+-	Run  the tutorial model
+-	Run the tool for different case studies than the one in the tutorial
+
+Before we get into the heart of the tutorial, it is important to make a few preliminaries for using the tool if you have a Windows operating system.
```

Comment:
It would be good to specify after here the scope of the short tutorial: provide a sample simple model that executes fast, but for full model representation and for realistic study we should refer to the standard model execution discussed in the customization basic

Reply:
_I could change this first part of the text like this, and keep only the reference to the tutorial case:_

The installation procedure installs PyPSA-Distribution model with all the software dependencies needed to build and run it. 

The purpose of this section is to make available to the user a simplified case study through which to become familiar with the code and its execution.

To this aim, we have chosen to present the simulation of a microgrid in Nigeria, choosing the tutorial mode within the configuration file. In this way, the user can explore the different possibilities of the tool, but at a lower expense of computational resources than in the case of a full simulation.


.. note::

    For realistic case studies it will be necessary to run the tool in full configuration and not in tutorial configuration. The tutorial mode is used only to become familiar with the tool.

Before we get into the heart of the tutorial, it is important to make a few preliminaries for using the tool if you have a Windows operating system.

Reply:
Agree, the note could be moved up kind-of right after the title